apps/services/sessions/infra/sessions.ts

```diff
@@ -0,0 +1,97 @@
+import { json, service, ServiceBuilder } from '../../../../infra/src/dsl/dsl'
+
+const namespace = 'sessions'
+const serviceName = 'services-sessions'
+const workerName = 'services-sessions-worker'
+const imageName = 'services-sessions'
+
+export const serviceSetup = (): ServiceBuilder<typeof serviceName> => {
+  return service(serviceName)
+    .namespace(namespace)
+    .image(imageName)
+    .postgres({
+      username: serviceName,
+      name: serviceName,
```

Comment:
@brynjarorng  the worker is also connecting to postgres, so the `workerSetup` must also get these two settings?